dali/operators/reader/loader/video/video_file_test.cc

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <exception>
+
+#include "dali/test/dali_test_config.h"
+#include "dali/core/error_handling.h"
+#include "dali/operators/reader/loader/video/video_test_base.h"
+#include "dali/operators/reader/loader/video/video_file.h"
+
+
+namespace dali {
+class VideoFileTest : public VideoTestBase {
+};
+
+
+TEST_F(VideoFileTest, ConstantFrameRate) {
```

Comment:
Does it make sense to loop through all the frames we have in the file and compare as a simple yet comprehensive test? Or it is just too long?

Reply:
I test this at the level above, with the tests of the reader. I didn't what to do it here, just to keep it short and simple (https://github.com/awolant/DALI/blob/video_reader_cpu/dali/operators/reader/video_reader_cpu_op_test.cc#L31).

Reply:
Sure